analyzers/Directory.Build.props

```diff
@@ -24,4 +24,13 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    <Deterministic>true</Deterministic>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <SourceRoot Include="$(MSBuildStartupDirectory)/"/>
```

Comment:
Needed. Otherwise we get the following error:
```
Microsoft.Managed.Core.targets(191,5): error : SourceRoot items must include at least one top-level (not nested) item when DeterministicSourcePaths is true
```